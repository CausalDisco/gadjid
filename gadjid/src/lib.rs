// SPDX-License-Identifier: MPL-2.0
#![warn(missing_docs)]
//! gadjid -  Graph Adjustment Identification Distance library
mod ascending_list_utils;
mod graph_loading;
pub mod graph_operations;
mod partially_directed_acyclic_graph;

pub use graph_loading::constructor::EdgelistIterator;
pub use partially_directed_acyclic_graph::LoadError;
pub use partially_directed_acyclic_graph::PDAG;
