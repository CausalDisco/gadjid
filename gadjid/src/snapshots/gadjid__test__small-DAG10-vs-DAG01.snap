---
source: src/lib.rs
expression: "test(g_true, g_guess)"
---
g_true: 20010.DAG-10
g_guess: 20001.DAG-10
ancestor_aid:
  - 0.25555555555555554
  - 23
oset_aid:
  - 0.26666666666666666
  - 24
parent_aid:
  - 0.25555555555555554
  - 23
shd:
  - 0.28888888888888886
  - 13
t:
  - 3
  - 4
  - 6
  - 7
  - 8
y: 2
z:
  - 0
  - 1
  - 5
  - 9
possible_descendants_of_t_in_g_guess:
  - 3
  - 4
  - 5
  - 6
  - 7
  - 8
  - 9
not_amenable_in_g_guess_wrt_t: []
proper_ancestors_of_y_in_g_guess_wrt_t:
  - 2
oset_for_t_onto_y_in_g_guess: []
not_validly_adjusted_for_in_g_guess_by_parents_of_t:
  - 0
  - 1
  - 2
  - 6
not_validly_adjusted_for_in_g_guess_by_oset_for_t_onto_y:
  - 0
  - 1
  - 2
  - 5
  - 9
not_validly_adjusted_for_in_g_guess_by_empty_set:
  - 0
  - 1
  - 2
  - 5
  - 9
not_validly_adjusted_for_in_g_guess_by_z:
  - 0
  - 1
  - 2
  - 5
  - 9
